Description: Disable the `nusb` backend
 This new backend requires the `nusb` crate which is not available in
 Debian yet. This patch should be dropped once `rust-nusb` is accepted
 in Debian.
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Update: 2024-11-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -41,11 +41,6 @@
 path = "examples/rockusb.rs"
 required-features = ["libusb"]
 
-[[example]]
-name = "rockusb-nusb"
-path = "examples/rockusb-nusb.rs"
-required-features = ["nusb"]
-
 [dependencies.bytes]
 version = "1.4.0"
 
@@ -55,17 +50,9 @@
 [dependencies.fastrand]
 version = "2"
 
-[dependencies.futures]
-version = "0.3.31"
-optional = true
-
 [dependencies.num_enum]
 version = "0.5"
 
-[dependencies.nusb]
-version = "0.1.10"
-optional = true
-
 [dependencies.rusb]
 version = "0.9.4"
 optional = true
@@ -80,7 +67,6 @@
 version = "0.4.5"
 features = [
     "gzip",
-    "futures-io",
 ]
 
 [dependencies.bmap-parser]
@@ -102,18 +88,6 @@
 [dependencies.rockfile]
 version = "0.1.2"
 
-[dependencies.tokio]
-version = "1.40.0"
-features = ["full"]
-
-[dependencies.tokio-util]
-version = "0.7.12"
-features = ["compat"]
-
 [features]
 default = ["libusb"]
 libusb = ["dep:rusb"]
-nusb = [
-    "dep:nusb",
-    "dep:futures",
-]
